app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -125,15 +128,31 @@ class ProfileChooserFragmentPresenter @Inject constructor(
     }
   }
 
+  private fun formatString(stringToFormat: String): String {
+    var stringToFormat = stringToFormat
+    if (stringToFormat.length > USER_NAME_MAX_CHAR_COUNT) {
+      stringToFormat = stringToFormat.substring(0, USER_NAME_MAX_CHAR_COUNT - 1) + "..."
```

Comment:
We should be ellipsizing this in the UI, e.g. via TextView's ellipsize method. That also performs smart ellipsizing to ensure things aren't broken awkwardly (more relevant for sentences, but still a good idea to generally use).

Reply:
@BenHenning  So lets keep 2 as max line and remove max length for the same ... Coz this was a requirement from designer doc.

Reply:
changed to 
maxLines =2
ellipsize=end